experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -278,8 +278,7 @@ function Base.show(io::IO, CPS::CoveredProjectiveScheme)
     end
     print(io, Lowercase(), base_scheme(CPS))
     if n != 0
-      print(io, " covered with $n projective patch")
-      n > 1 && print(io, "es")
+      print(io, " covered with ", ItemQuantity(n, "projective patch", "projective patches"))
```

Comment:
Can be simplified once https://github.com/Nemocas/AbstractAlgebra.jl/pull/1460 lands and is released to
```suggestion
      print(io, " covered with ", ItemQuantity(n, "projective patch"))
```

Reply:
Ok so if I push the changes some doctests will fail but then we re-run everything once the new Nemo is here ?

Reply:
New AA, actually. Basically, just change the `Project.toml` of Oscar to require AA 0.32.5 and add that to this PR, then push it once https://github.com/JuliaRegistries/General/pull/92810 is merged (probably in 15-20 minutes)